Cargo.toml

```diff
@@ -8,11 +8,12 @@ name = "rusl"
 crate-type = ["staticlib", "dylib"]
 
 [dependencies]
-lazy_static = { version = "0.2.1", features = ["spin_no_std"] }
+lazy_static = { version = "0.2.2", features = ["spin_no_std"] }
 rlibc = "1.0.0"
-spin = "0.4.0"
-syscall = "0.2.1"
-va_list = "0.0.3"
+spin = "0.4.5"
+sc = "0.1.5"
+va_list = { version = "0.1.0", features = ["no_std"] }
+compiler_builtins = { git = "https://github.com/rust-lang-nursery/compiler-builtins" }
```

Comment:
Can you specify a `rev` key here so that we're not only relying on Cargo.lock to pin the correct version? Makes it so it's easy to upgrade this dep without editing or deleting the lockfile.